### PR TITLE
Move kimchi tests to dynamic feature modules

### PR DESCRIPTION
**Pull request by TomMarshBBC**

### CODE REVIEW

File: bottom/src/androidTest/java/bbc/co/uk/nautilusapp/bottomnav/ui/kimchi/ScreenAssertions.kt [hunk 1]
```diff
@@ -28,19 +23,11 @@ fun UserAssertions.bottomNav() = apply {
...
-
 fun UserAssertions.withContent() = apply {
-    onRecyclerViewAtPosition(R.id.indexList, 0).assertHasDescendants(
-        withId(R.id.content_card_title),
-        withId(R.id.content_card_link),
-        withId(R.id.content_card_info)
+    onRecyclerViewAtPosition(uk.co.bbc.rubik.indexui.R.id.indexList, 0).assertHasDescendants(
+        withId(uk.co.bbc.cubit.R.id.content_card_title),
+        withId(uk.co.bbc.cubit.R.id.content_card_link),
+        withId(uk.co.bbc.cubit.R.id.content_card_info)
```
Comment (geofffalk) [inline bottom/src/androidTest/java/bbc/co/uk/nautilusapp/bottomnav/ui/kimchi/ScreenAssertions.kt]: why are these needed now? Are you trying to avoid namespace conflicts?

Comment (geofffalk) [reply bottom/src/androidTest/java/bbc/co/uk/nautilusapp/bottomnav/ui/kimchi/ScreenAssertions.kt]: just run tests, i see they are required now

Comment (TomMarshBBC) [reply bottom/src/androidTest/java/bbc/co/uk/nautilusapp/bottomnav/ui/kimchi/ScreenAssertions.kt]: Yeah, it would complain about not being able to find the resources.

btw if you didn't know, there is no tooling for instrumentation tests in Dynamic Feature Modules at the moment, so you have to use the command line I.E. `./gradlew bottom:connectedAndroidTest --no-parallel` etc..
